src/datepicker.ts

```diff
@@ -191,6 +191,10 @@ export class Datepicker extends LitElement {
       /* outline: none; */
     }
 
+    [dir="rtl"].calendars-container, .calendars-container[dir="rtl"] {
```

Comment:
This can already merged with the one at https://github.com/motss/app-datepicker/pull/194/files#r578344066